core/vtk/ttkPointSetToCurve/ttkPointSetToCurve.h

```diff
@@ -0,0 +1,53 @@
+/// \class ttkPointSetToCurve
+/// \ingroup vtk
+/// \author Jonas Lukasczyk <jl@jluk.de>
```

Comment:
you might want to put your own contact info instead.

Reply:
Fixed now